zerver/views/invite.py

```diff
@@ -93,7 +96,14 @@ def invite_users_backend(
 
     if len(streams) and not user_profile.can_subscribe_other_users():
         raise JsonableError(_("You do not have permission to subscribe other users to channels."))
-
+    
+    if is_guest_invite:
+        visible_users_count = sum(len(stream.recipient.get_users()) for stream in streams)
+        # Construct a message to be sent to the frontend
+        message = f"Guests will be able to see {visible_users_count} users in their channels when they join."
```

Comment:
This can't possibly be correct, because some users could be in more than one of those channels. 

Additionally, we need to be able to ask this question before actually creating the invites. The implementation should be possible to do entirely client-side in the web app, without talking to the server, since the web app currently caches the relevant subscribe metadata in the `peer_data` module.

Finally, you need translation tags: https://zulip.readthedocs.io/en/latest/translating/internationalization.html

Reply:
ok , can you explain in more details , will go through the docs once more
and the link mentioned